### PR TITLE
disable wireit cache on PR pipelines

### PR DESCRIPTION
**Pull request by Mearman**
quick temporary fix to https://github.com/breadboard-ai/breadboard/issues/629

### CODE REVIEW

**Review by aomarks — APPROVED** [review]

File: .github/workflows/labs-prototypes-ci.yml [hunk 1]
```diff
@@ -27,7 +27,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "pip"
       - run: pip install packages/breadboard-python
+
       - uses: google/wireit@setup-github-actions-caching/v1
+        if: github.event_name != 'pull_request'
```
Comment (aomarks) [inline .github/workflows/labs-prototypes-ci.yml:32]: Mind adding a comment here, linking to the other issue?

**Review by aomarks — CHANGES_REQUESTED** [review]

File: .github/workflows/labs-prototypes-ci.yml (hunk 1 above)
Comment (Mearman) [reply .github/workflows/labs-prototypes-ci.yml:32]: a comment in the yaml?
this issue? https://github.com/breadboard-ai/breadboard/issues/629

Comment (aomarks) [reply .github/workflows/labs-prototypes-ci.yml:32]: Yeah, so that the next time somebody looks at that line, they'll know why it's there.

Comment (Mearman) [reply .github/workflows/labs-prototypes-ci.yml:32]:
```suggestion
        if: github.event_name != 'pull_request' # For PRs, caching is disabled to ensure a clean build
```

**Review by aomarks — APPROVED** [review]

**Pull request merged**